### PR TITLE
Add instruction on how to contribute

### LINKED ISSUE #9: Add How to contribute guide

### PR DESCRIPTION
**Pull request by skoehnen**
Closes #9 

I decided to add a separate document for the contribution guidelines because the file CONTRIBUTING.md is automatically linked when creating a new pull request ;)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+This repository is a meta repository to bring together the different metadata
+schema collections and maintaining the python code-base.
+
+If you want to contribute to the schemas themselves please use the repository
```
Comment (lzehl) [inline CONTRIBUTING.md]: ... please do so in the...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- [**openMINDS_controlledTerms**](https://github.com/HumanBrainProject/openMINDS_controlledTerms)
+
+If you want to contribute to this repository, feel free to fork it and open a
+pull request with your changes.
+You are also welcome to contribute to discussion in the issue tracker and of
+course to report issues you are facing yourself.
+
+Another important contribution is to spread the word about this project.
+
+If you find our project helpful you can reference it in articles,
```
Comment (lzehl) [inline CONTRIBUTING.md:16]: I would delete this sentence

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+posters, etc.
+
+You can also star the project to show us that you are using it.
+
+To get in contact with us, you can also reach us with openminds@ebrains.eu
```
Comment (lzehl) [inline CONTRIBUTING.md]: delete "also"

File: README.md [hunk 4]
```diff
@@ -43,5 +43,8 @@ Currently, the following generators are available:
 - *generate_python.py*
 - *generate_html.py*
 
+## How to contribute
```
Comment (lzehl) [inline README.md:75]: The doc is not that big... I would integrate it into the README directly for now.

**Review by lzehl — CHANGES_REQUESTED** [review]
I would integrate the how to contribute directly into the readme. 
It's not that long and can be shortened even a bit more.

File: README.md (hunk 4 above)
Comment (skoehnen) [reply README.md:75]: The advantage of a dedicated CONTRIBUTING document is that it is integrated in the github process to create a pull request.

Comment (lzehl) [reply README.md:75]: Never mind this request then. I overlooked that part in your PR description. Thanks

File: CONTRIBUTING.md (hunk 3 above)
Comment (skoehnen) [reply CONTRIBUTING.md]: Done 👍

File: CONTRIBUTING.md (hunk 2 above)
Comment (skoehnen) [reply CONTRIBUTING.md:16]: Done :+1:

Comment (skoehnen) [reply CONTRIBUTING.md:16]: @lzehl did mean the sentence in line 16?

File: CONTRIBUTING.md (hunk 1 above)
Comment (skoehnen) [reply CONTRIBUTING.md]: Done 👍

**Pull request merged**